test/SlowTests/Issues/RavenDB-13987.cs

```diff
@@ -0,0 +1,154 @@
+            {
+                var doc = new DatabaseRecord(_databaseName);
+                databaseResult = await store.Maintenance.Server.SendAsync(new CreateDatabaseOperation(doc, clusterSize));
+            }
+
+            Assert.Equal(clusterSize, databaseResult.Topology.AllNodes.Count());
+
+            foreach (var server in Servers)
+            {
+                await server.ServerStore.Cluster.WaitForIndexNotification(databaseResult.RaftCommandIndex);
+            }
+
+            foreach (var server in Servers.Where(s => databaseResult.NodesAddedTo.Any(n => n == s.WebUrl)))
+            {
+                await server.ServerStore.DatabasesLandlord.TryGetOrCreateResourceStore(_databaseName);
+            }
+
+            var rnd = new Random();
+            var index = rnd.Next(0, Servers.Count - 1);
+            using (var store = new DocumentStore { Urls = new[] { Servers[index].WebUrl }, Database = _databaseName }.Initialize())
+            {
+                using (var s = store.OpenAsyncSession())
+                {
+                    await s.StoreAsync(new User() { Name = "Egor" }, "foo/bar");
+                    await s.SaveChangesAsync();
+                }
+
+                // wait until all dbs become idle
+                var now = DateTime.Now;
+                var nextNow = now + TimeSpan.FromSeconds(300);
+                while (now < nextNow && GetIdleCount() < 3)
+                {
+                    Thread.Sleep(3000);
+                    now = DateTime.Now;
+                }
+
+                Assert.Equal(3, GetIdleCount());
+
+                var putConfiguration = new ServerWideBackupConfiguration
+                {
+                    FullBackupFrequency = "*/2 * * * *",
+                    IncrementalBackupFrequency = "*/2 * * * *",
+                    LocalSettings = new LocalSettings { FolderPath = backupPath },
+                };
+
+                var result = await store.Maintenance.Server.SendAsync(new PutServerWideBackupConfigurationOperation(putConfiguration));
+                var serverWideConfiguration = await store.Maintenance.Server.SendAsync(new GetServerWideBackupConfigurationOperation(result.Name));
+                Assert.NotNull(serverWideConfiguration);
+
+                var record1 = await store.Maintenance.Server.SendAsync(new GetDatabaseRecordOperation(store.Database));
+                var backups1 = record1.PeriodicBackups;
+                _taskId = backups1.First().TaskId;
+
+                Assert.Equal(1, backups1.Count);
+            }
+
+            Assert.Equal(3, GetIdleCount());
+
+            // wait for the backup occurrence
+            var idleCount = WaitForCount(_reasonableWaitTime, 2, GetIdleCount);
+            Assert.Equal(2, idleCount);
+
+            // backup status should not wakeup dbs
+            var count = WaitForCount(_reasonableWaitTime, 3, CountOfBackupStatus);
+            Assert.Equal(3, count);
```

Comment:
Can we add to this test that database will be waken up after all the modifications and backups will be performed?